src/PageContent/RequestPickup/RecurringPickupRequest.js

```diff
@@ -148,7 +151,7 @@ class RecurringPickupRequest extends Component {
         if (!this.handleValidation()) {
             alert('Form has errors');
         } else {
-            let dateTimeStringToTimestamp = (dateString, timeString) => Date.parse(dateString + ' ' + timeString);
+            let dateTimeStringToTimestamp = (dateString, timeString) => Date.parse(dateString.replace(/-/g, '/') + ' ' + timeString);
```

Comment:
(Safari doesn't support creating `Date`s from `"2018-02-12"` format)

Reply:
yay safariiii

Reply:
Since we are using moment, we could also use moment for this next time

Reply:
good call, i'll change it